docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1258,6 +1258,41 @@ Renames an existing field. If the field doesn't exist or the new name is already
 }
 --------------------------------------------------
 
+[[script-processor]]
+=== Script Processor
+
+Script Processor allows inline, stored, and file scripts to be executed within Ingest Pipelines.
+
+Scripts use Elasticsearch's Script Service to compile the processor's script for each document. This
```

Comment:
Maybe just refer to this page (https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-using.html#_script_parameters) and add something like this:

[IMPORTANT]
.Script caching
========================================
The [use of script parameters](https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-using.html#_script_parameters) for caching and avoiding that scripts need to be compiled for each request.

We can then add the `script.cache.max_size` setting to that page.